v2/internal/testcommon/kube_per_test_context.go

```diff
@@ -561,6 +554,16 @@ func (tc *KubePerTestContext) ExpectSecretHasKeys(name string, expectedKeys ...s
 	}
 }
 
+func (tc *KubePerTestContext) CreateTestNamespaces(names ...string) {
+	for _, name := range names {
+		tc.CreateResourceUntracked(&corev1.Namespace{
```

Comment:
Call `createTestNamespace` in here rather than duplicating its functionality?

Reply:
Though actually looking at `createTestNamespace` it's written a little weird. Still think it's worth reusing it rather than not though